src/transaction-hooks.js

```diff
@@ -0,0 +1,42 @@
+const debug = require('debug')('feathers-knex-transaction');
```

Comment:
We may want to remove this file to just `hooks` and call the methods `transactionStart`, `transactionEnd` and `transactionRollback` and export it as `init.hooks` instead of https://github.com/feathersjs/feathers-knex/pull/113/files#diff-1fdf421c05c1140f6d71444ea2b27638R339